### PR TITLE
update readme file with example usage for mounting multiple versions

### PR DESCRIPTION
**Pull request by przbadu**
related to: https://github.com/ruby-grape/grape/issues/1305#issuecomment-193223103.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@
   - [Header](#header)
   - [Accept-Version Header](#accept-version-header)
   - [Param](#param)
+  - [Mounting multiple versions](#mounting-multiple-versions)
```
Comment (dblock) [inline README.md]: Capitalize Multiple Versions

File: README.md [hunk 2]
```diff
@@ -390,6 +391,68 @@ version 'v1', using: :param, parameter: 'v'
...
+    |-- v2/
+    |--- api.rb
+
+
+`api/api.rb`
+
+```ruby
+class API < Grape::API
+  prefix          'api'
+  version         ['v2', 'v1']
```
Comment (dblock) [inline README.md]: This is incorrect though, no? It doesn't do anything as far as I know.

File: README.md [hunk 3]
```diff
@@ -390,6 +391,68 @@ version 'v1', using: :param, parameter: 'v'
...
+  mount V1::API
+end
+```
+
+`api/v1/api.rb`
+
+```ruby
+class V1::API < Grape::API
+  version 'v1' # required block although `version ['v2', 'v1']` included in api.rb
+
```
Comment (dblock) [inline README.md]: Remove one of the empty lines.

**Comment by dblock** [comment]
You might also want to check http://code.dblock.org/2013/07/19/evolving-apis-using-grape-api-versioning.html to improve content here.

**Comment by dblock** [comment]
Squash your commits, too, please.

**Comment by przbadu** [comment]
@dblock: I will improve it this weekend

**Comment by dblock** [comment]
Bump @przbadu ;)

**Comment by przbadu** [comment]
please refer to https://github.com/ruby-grape/grape/pull/1346

**Pull request closed without merging**